docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -14,19 +14,69 @@ helpviewer_keywords:
+| Deserialize inferred type to `object` properties      | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#deserialization-of-object-properties) |
+| `DateTimeZoneHandling`, `DateFormatString` settings   | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#specify-date-format) |
+| Deserialize strings as numbers                        | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#quoted-numbers) |
+| Deserialize `Dictionary` with non-string key          | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#dictionary-with-non-string-key) |
+| Polymorphic serialization                             | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#polymorphic-serialization) |
+| Polymorphic deserialization                           | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#polymorphic-deserialization) |
+| `Required` setting on [JsonProperty] attribute        | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#required-properties) |
+| Deserialize JSON `null` literal to non-nullable types | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#deserialize-null-to-non-nullable-type) |
+| Deserialize to immutable classes and structs          | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#deserialize-to-immutable-classes-and-structs) |
+| `[JsonConstructor]` attribute                         | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#specify-constructor-to-use) |
+| `DefaultContractResolver` to exclude properties       | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#conditionally-ignore-a-property) |
+| `NullValueHandling` setting on `[JsonProperty]` attribute | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#conditionally-ignore-a-property)  |
+| `DefaultValueHandling` setting on `[JsonProperty]` attribute | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#conditionally-ignore-a-property)  |
+| `DefaultValueHandling` global setting                 | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#conditionally-ignore-a-property) |
+| Callbacks                                             | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#callbacks) |
+| Support for public and non-public fields              | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround](#public-and-non-public-fields) |
```

Comment:
The ordering in this section makes sense. The only thing is `Support for public and non-public fields` is one of the most common things people see and the way the table is structure, it has lower visibility.

I believe this is the only case that has this issue, so I think the current layout is the best. I guess, once/if we have samples, we can move it up to the top of the "Not supported, workaround, sample" section.